src/pages/components/combobox.mdx

```diff
@@ -0,0 +1,233 @@
+---
+title: Combobox
+description: Combobox is an input plus a list box that appears beneath it. It searches and filters down options for the user.
+package: '@zendeskgarden/react-dropdowns'
+components:
+  - dropdowns/src/elements/Combobox/Combobox.tsx
+  - dropdowns/src/elements/Dropdown/Dropdown.tsx
+  - dropdowns/src/elements/Fields/Field.tsx
+  - dropdowns/src/elements/Fields/Hint.tsx
+  - dropdowns/src/elements/Menu/Items/Item.tsx
+  - dropdowns/src/elements/Fields/Label.tsx
+  - dropdowns/src/elements/Menu/Menu.tsx
+---
+
+<Usage>
+  <Use>
+    <ul>
+      <li>To filter down a list of options</li>
+      <li>To quickly find a known option</li>
+      <li>As a search field</li>
+    </ul>
+  </Use>
+  <Misuse>
+    <ul>
+      <li>
+        <Markdown>
+          For a searchable dropdown, use [Autocomplete](/components/autocomplete) instead
```

Comment:
Well it would be considered a dropdown and it is searchable.. unlike the `Select`, `Multiselect` and `Autocomplete`, the Combobox doesn't have a chevron to drop down a menu with this component. The combobox is amorphous since it technically can be a `Multiselect` and `Autocomplete` [according to the spec](https://www.w3.org/TR/wai-aria-practices/examples/combobox/aria1.1pattern/listbox-combo.html).

Do you think we should change this?

Reply:
I see. I don't have a suggestion at this time. But maybe we can revisit this if consumers need clarification.

Reply:
In a reconsidered world, `Combobox` would've been the `Dropdown` baseline and `Autocomplete`, `Multiselect`, and (maybe) `Select` would've inherited from that flexible baseline. With this inheritance in mind, all three of `Combobox` (a no-select), `Autocomplete` (a single select), and `Multiselect` (erm multi-select) are searchable. And all three drop down a menu. So I'm kind of tripping on the wording here as well.

Reply:
Mike is correct that the combobox is not a dropdown because it does not have a chevron to show a menu of items. Looking at the components within the Dropdown group today, they all have this. Combobox does not.

Therefore it's an issue to have Combobox in the Dropdown group--it's incorrect and confusing. For now, I think the simplest path is to have Combobox not be under the Dropdowns group and on a separate page of its own. That way it makes sense while we don't have to overhaul the entire information architecture. Thoughts?

Sorry I'm giving this kind of feedback now. I was not aware of where we were placing this component page. The rest of the content looks good. Thanks Mike!

cc: @bethanybeech

Reply:
Does the absence of a chevron icon indicate `Combobox` is not a dropdown? I don't think the word has been defined in Garden, so I'd be curious of how the team defines this. My definition is that a dropdown is a widget that opens or expands when it is selected, revealing a list of items. Although Garden's combo box does not have an icon, it is optional according to ARIA:

> Many implementations also include a third optional element -- a graphical button adjacent to the textbox, indicating the availability of the popup.

Source: https://www.w3.org/TR/wai-aria-practices/#combobox

In Garden, there's a `Dropdown` component that manages the behavior and state of components that belong to the "dropdown family" ([select](https://www.w3.org/TR/wai-aria-practices/examples/listbox/listbox-collapsible.html), [combobox](https://www.w3.org/TR/wai-aria-practices/examples/combobox/aria1.1pattern/listbox-combo.html), [menu button](https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-links.html)). Note the similarities of each:

```jsx
// dropdown manages state for `Combobox`
<Dropdown>
  <Field>
    <Label>Choose a vegetable</Label>
    <Combobox placeholder="Search vegetables" />
  </Field>
  <Menu>
    <Item>Spinach</Item>
    <Item>Kale</Item>
    <Item>Lettuce</Item>
  </Menu>
</Dropdown>
```

```jsx
// dropdown manages state for `Autocomplete`
<Dropdown>
  <Field>
    <Label>Choose a vegetable</Label>
    <Autocomplete>Spinach</Autocomplete>
  </Field>
  <Menu>
    <Item>Spinach</Item>
    <Item>Kale</Item>
    <Item>Lettuce</Item>
  </Menu>
</Dropdown>
```

```jsx
// dropdown manages state for `Select`
<Dropdown>
  <Field>
    <Label>Choose a vegetable</Label>
    <Select>Spinach</Select>
  </Field>
  <Menu>
    <Item>Spinach</Item>
    <Item>Kale</Item>
    <Item>Lettuce</Item>
  </Menu>
</Dropdown>
```

```jsx
// dropdown manages state for Menu`
<Dropdown>
  <Trigger>
    <Button>
      Choose vegetable
      <Button.EndIcon isRotated={rotated}>
        <ChevronIcon />
      </Button.EndIcon>
    </Button>
  </Trigger>
  <Menu>
    <Item>Spinach</Item>
    <Item>Kale</Item>
    <Item>Lettuce</Item>
  </Menu>
</Dropdown>
```

These reasons lead me to believe that `Combobox` is a dropdown and belongs in the `react-dropdowns` package. 

It seems like components on the website don't necessarily have to be grouped by their packages. For example, `Stepper` and `Timeline` are part of the accordion 🪗 package, but are visible at the root level of the side panel. My understanding is that this setup highlights the `Stepper` and `Timeline` as significant in itself. I think `Combobox` can be visible at the root level if it is significant in itself, or makes things more clear for consumers.

tldr: I think `Combobox` is a dropdown but can probably still live at the root level (if it helps make things more clear).

Reply:
>  It seems like components on the website don't necessarily have to be grouped by their packages. For example, Stepper and Timeline are part of the accordion 🪗 package, but are visible at the root level of the side panel. My understanding is that this setup highlights the Stepper and Timeline as significant in itself. I think Combobox can be visible at the root level if it is significant in itself, or makes things more clear for consumers.

Well said @hzhu I think it would be worthwhile for Combobox to also be at the root for clarity.

To answer your question, my understanding of what a dropdown is informed by other design systems which is why I think the chevron is a distinction, not saying that's the correct definition or that any of these other design systems have it right:
https://atlassian.design/components/dropdown-menu/examples
https://clarity.design/angular-components/dropdown/
https://material.io/components/menus#dropdown-menu
https://design.cms.gov/components/dropdown/

Thinking on this more, I believe it's less about the chevron to your point and ARIA, and more about the interaction. 

The difference between combobox and the other "Dropdown" components for Garden is very small as a reader of the documentation so that is a challenge here. But a combobox has a listbox that appears as you type in the input whereas autocomplete has dropdown that appears when the user clicks into the input. User has to click to get into the box for components Menu, Select, and Multiselect.

Reply:
If it makes things more clear @diana-oum , let's discuss this deeper because I do have some concerns with what @hzhu & @jzempel brought up. Does dropdown mean having a chevron to fit the bill or is it any component that drops a list of items? Another concern of mine is convention, and changing it would contrast how we've structured the project, which is something to consider..

I don't mind making the change if we agree it would clarify things and makes sense, and we've done it in the past as @hzhu pointed out. Ultimately, how should we define Dropdown?

Reply:
Another thought I had goes back to the phrase "searchable dropdown". Do you think we should rephrase it more specifically? Maybe "searchable and selectable dropdown"?

Reply:
@Francois-Esquire I agree, this is a conversation that really should have required us earlier in the process for all of us to come together and chat about. It's a large decision for sure. But since we're trying to wrap up Q4 things ASAP, let's just keep this learning in mind when working on future component pages. 

>  Another thought I had goes back to the phrase "searchable dropdown". Do you think we should rephrase it more specifically? Maybe "searchable and selectable dropdown" could remove some confusion?

That is something to consider and one that I'm gonna try to push for instead to keep the convention we have now to get us moving forward. Let me brainstorm with Bethany as she was the driver of the documentation.

Reply:
@Francois-Esquire - I jammed with Bethany on this just now. Let's replace that first bullet in the Not for this section with this sentence: "To see a list of options in the dropdown without entering text in the input field, use Autocomplete" We'll maintain the concept model that's already established while trying to be clear as we can.

Based on this thread and our own confusion, there are terminology issues here. But that needs to be addressed in a separate initiative where we all can be involved from the beginning! Let's also learn from our consumers like Henry already mentioned. Appreciate everyone's input here. Thanks!

Reply:
@diana-oum let's take this offline since there are other things to consider.